101-cosmosdb-mongodb/metadata.json

```diff
@@ -5,7 +5,7 @@
   "description": "This template creates an Azure Cosmos DB account for MongoDB API in two regions using shared database throughput with two collections.",
```

Comment:
Can you change the description and summary to be.

This template creates an Azure Cosmos DB account for MongoDB API (3.2 or 3.6) in two regions using shared database throughput with two collections.